src/components/TabLink/TabLink.tsx

```diff
@@ -0,0 +1,24 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { tabs } from '../../api/tabs';
+import { PageNavLink } from '../PageNavLink';
+
+type Props = {
+  selectedTabId: string,
+};
+
+export const TabLink: FC<Props> = ({ selectedTabId }) => (
+  <div className="tabs is-boxed">
+    <ul>
+      {tabs.map(tab => (
+        <li
+          data-cy="Tab"
+          key={tab.id}
+          className={classNames({ 'is-active': tab.id === selectedTabId })}
+        >
+          <PageNavLink to={`/tabs/${tab.id}`} text={tab.title} />
```

Comment:
You're using `NavLink` for tabs and applying 'is-active' class on the NavLink and on the `li` element, you need to make a separate `Link` for tabs and apply class 'is-active' only on `li` element

Reply:
agree